### PR TITLE
[Messenger] Add caution message about messenger consume command and profiler

### PR DESCRIPTION
**Pull request by barell**
Related to https://github.com/symfony/symfony/issues/53637

### CODE REVIEW

File: console.rst [hunk 1]
```diff
@@ -641,6 +641,14 @@ profile is accessible through the web page of the profiler.
     terminal supports links). If you run it in debug verbosity (``-vvv``) you'll
     also see the time and memory consumed by the command.
 
+.. caution::
+
+    If you use Messenger component and want to profile the ``messenger:consume``
+    command please be aware that it will only create the profile only when
```
Comment (94noni) [inline console.rst:647]: 2 "only" in same sentence seem unatural to me (I am non native English speaker)

what about:

please be aware that the profile will only be done with the --no-reset option

**Review by 94noni — APPROVED** [review]
nice addition 👍🏻

File: console.rst (hunk 1 above)
Comment (javiereguiluz) [reply console.rst:647]: We did this reword while merging. Thanks.

**Comment by javiereguiluz** [comment]
Bartosz, thanks for this ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**